docs/buildah-run.1.md

```diff
@@ -151,6 +153,8 @@ Current supported mount TYPES are bind, cache, secret and tmpfs. <sup>[[1]](#Foo
 
               · gid: gid for cache directory.
 
+              · from: image name for the root of the source. Defaults to host cache directory.
```

Comment:
I was unable to add this in following PR, as PR is only scoped to `buildkit` so as of now it can only mount images and stages. Mounting container seems like a `buildah` specific feature so I'll do that in a new PR if that works.